app/src/main/java/mozilla/lockbox/store/GleanTelemetryStore.kt

```diff
@@ -43,5 +45,9 @@ class GleanTelemetryStore(
             }
             .addTo(compositeDisposable)
         gleanWrapper.initialize(context, BuildConfig.BUILD_TYPE)
+
+        // Get legacy telemetry ID
+        LegacyIds.clientId.generateAndSet() // Generate a new UUID and record it
+        LegacyIds.clientId.set(UUID.randomUUID()) // Set a UUID explicitly
```

Comment:
This generates a new random id, and then does it again on the next line.  I think you need to do something like `LegacyIds.clientId.set(TelemetryHolder.get().getClientId())`

Reply:
It looks like the `TelemetryHolder` gets set in Lockwise [here](https://github.com/mozilla-lockwise/lockwise-android/blob/096cdac7181f70c040f60eb19175e1455de25537/app/src/main/java/mozilla/lockbox/store/TelemetryStore.kt#L58), and you may need to `import` it to access it.